bin/build-stub

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env php
```

Comment:
I opened an issue https://github.com/humbug/box/issues/206 to be able to remove the main script since your stub ditches it. It's however a minor optimization that shouldn't be blocking